### ISSUE TITLE
Windscribe disconnects and loops when hotspot is enabled (Windows & Ubuntu)

### ISSUE THREAD

**Issue by Paraguanads** [Windscribe/Desktop-App#175]
**Describe the bug**
When creating a 2.4 GHz Wi-Fi hotspot on a desktop (tested on both Windows 10 and Ubuntu 24.04) and connecting an older laptop to this hotspot while running Windscribe VPN, the VPN client crashes and enters a connection loop. The VPN keeps attempting to reconnect until the hotspot is disabled. This issue seems specific to scenarios involving both hotspot functionality and Windscribe.

**OS and app information:**
OS: Windows 10 22H2 (Compilation 19045.4529), Ubuntu 24.04 LTS
App version: Windscribe 2.10.16 (Windows), Windscribe 2.10.15 (Linux)

**To Reproduce**

1. On the desktop, enable a Wi-Fi hotspot (using 2.4 GHz) while connected to Windscribe VPN.
2. On an older laptop, connect to the hotspot network.
3. Observe that the Windscribe client on the desktop crashes and begins looping attempts to reconnect until the hotspot is turned off.

**Expected behavior**
The VPN client should maintain a stable connection while the hotspot is active, allowing the connected devices to access the internet via the VPN tunnel without crashes or looping. If the Firewall switch is off this could cause data leakage on the host.

**Screenshots**
N/A

**Additional context**
I have no idea if this issue is presented in previous versions of Windscribe with similar network setups, though it’s the first time I’m using a hotspot with an older laptop in combination with Windscribe. The bug occurs across both Windows and Linux desktop setups, suggesting it might be related to how Windscribe handles hotspot-related network interfaces.

**Comment by jaxu** [comment]
What protocols does this occur on?  We are aware that WireGuard has some issues with hotspots, but IKEv2/OpenVPN may work.

**Comment by Paraguanads** [comment]
> What protocols does this occur on? We are aware that WireGuard has some issues with hotspots, but IKEv2/OpenVPN may work.

Indeed, I am using Wireguard on both devices. I just tried with the other protocols and was able to use the hotspot. The error seems to be related to Wireguard, as you indicate.
Thanks for replying!

**Comment by jaxu** [comment]
Thanks for checking!  As mentioned before, we are aware of this issue with WireGuard though it has been somewhat low priority.  I'll bump the priority on it and hopefully we'll get a chance to take a look in the near future.
